mwe_condition_working.jl

```diff
@@ -0,0 +1,76 @@
+using Oceananigans
+using Oceananigans.Units
+using ClimaOcean
+includet("BasinMask.jl")
+using .BasinMask
+
+Nx = Integer(360/4)
+Ny = Integer(180/4)
+Nz = Integer(100/4)
+
+arch = CPU()
+z_faces = (-4000, 0)
+
+underlying_grid = TripolarGrid(arch;
+                    size = (Nx, Ny, Nz),
+                    z = z_faces,
+                    halo = (5, 5, 4),
+                    first_pole_longitude = 70,
+                    north_poles_latitude = 55)
+
+@info "Defining bottom bathymetry"
+
+@time bottom_height = regrid_bathymetry(underlying_grid;
+                                    minimum_depth = 10,
+                                    interpolation_passes = 75, # 75 interpolation passes smooth the bathymetry near Florida so that the Gulf Stream is able to flow
+                                    major_basins = 2)
+
+# For this bathymetry at this horizontal resolution we need to manually open the Gibraltar strait.
+# view(bottom_height, 102:103, 124, 1) .= -400
+
+@info "Defining grid"
+
+@time grid = ImmersedBoundaryGrid(underlying_grid, GridFittedBottom(bottom_height); active_cells_map=true)
+                    
+@info grid           
+
+free_surface = SplitExplicitFreeSurface(grid; substeps=30)
+model = HydrostaticFreeSurfaceModel(; grid, free_surface)
+
+
+simulation = Simulation(model, Δt=0.001, stop_iteration=2000)
+
+function integrate_tuple(outputs; volmask, dims, condition, suffix::AbstractString) # Add suffix kwarg
+    for key in keys(outputs)
+        @info key
+        int_model_outputs = Integral(outputs[key]; dims, condition)
+        # int_model_outputs = NamedTuple((Symbol(string(key) * suffix) => Integral(outputs[key]; dims, condition)))
+    end
+    @info "I am doing dV now"
+    dV_int = NamedTuple{(Symbol(:dV, suffix),)}((Integral(volmask; dims, condition),))
+    # @info "I am merging now"
+    # int_outputs = merge(int_model_outputs, dV_int)
+    return nothing #int_model_outputs#int_outputs
```

Comment:
this seems like debugging? happy to merge it if this is how you like it 

see also https://github.com/taimoorsohail/ocean-ensembles/issues/29 which seems related

Reply:
I want to remove these files from the PR but I am not sure how... can we discuss?

Reply:
we can for sure.